pm/recipient.go

```diff
@@ -363,16 +366,24 @@ func (r *recipient) updateSenderNonce(rand *big.Int, ticket *Ticket) error {
 	defer r.senderNoncesLock.Unlock()
 
 	randStr := rand.String()
-	sn, ok := r.senderNonces[randStr]
-	if ok && ticket.SenderNonce <= sn.nonce {
-		return errors.Errorf("invalid ticket senderNonce sender=%v nonce=%v highest=%v", ticket.Sender.Hex(), ticket.SenderNonce, sn.nonce)
+	senderNonce, randKeySeen := r.senderNonces[randStr]
```

Comment:
```suggestion
	senderNonces, randKeySeen := r.senderNonces[randStr]
```

Nit: The plural feels a bit clearer here since the map is tracking a collection of nonces as opposed to a single one.